index.html

```diff
@@ -592,6 +592,34 @@ <h2>A project for first-time contributions</h2>
         </div>
          <!-- ________ Daniella's card END ________  -->
 
+        <!-- ________ McDaniels' card START ________  -->
+        <div class="card">
+        
+            <p class="name">McDaniels Okoro</p>
+            <p class="contact">
+                <i class="fa fa-twitter"></i>
+                <a href="http://www.twitter.com/acodeaholic" target="_blank">@acodeaholic</a>
+            </p>
+            <p class="about">I am a codeaholic, which means I love to code. Few years ago, I was allergic to coding, but now I aim to become a Fullstack
+                Developer. I love to code, because I intend to save the world one code at a time ;) My pet name is Jukels. Don't
+                ask what it means, because even I don't know :P</p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li>
+                        <a href="#" target="_blank" title="https://learn.fullstackacademy.com">Become a Millionaire with Fullstack Academy</a>
+                    </li>
```

Comment:
You need to move the `url` from `title` to `href`. Replace the `#` with the link. In `title` you can add a short description of the resource which will appear on hover, or simply leave it blank. 
You will need to do the same for the 3 resources links.
Thank you